docs/app/data-model/fields/text-numbers.md

```diff
@@ -31,6 +31,18 @@ A search input that will populate dropdown choices by making a request to a give
 - **Trigger**: `Throttle`, `Debounce` - The method used to trigger the web request as you type a query.
 - **Rate**: The delay in `milliseconds` used in the Trigger function.
 
+## Block Editor
+
+![A Block Editor allows users to create and edit content using blocks or modules](https://marketing.directus.app/assets/f631a2e1-cb27-434a-939b-eb15132ac46a.png)
```

Comment:
I feel like this would be a more appropriate alt text
```suggestion
![Showcase of the block editor with example blocks](https://marketing.directus.app/assets/f631a2e1-cb27-434a-939b-eb15132ac46a.png)
```